docs/how-to/deploy_charmed_magma_orchestrator.md

```diff
@@ -0,0 +1,86 @@
+# Deploy Charmed Magma Orchestrator
+
+## Requirements
+
+The Access Gateway must be installed on a Kubernetes cluster with the following specifications:
+
+- **:material-kubernetes: Kubernetes**: A cluster with a minimum of 3 Nodes with 2 vCPUs and 8 GB of RAM each.
```

Comment:
I would probably not say a minimum of 3 nodes, unless we specify requirements for production. It runs perfectly fine on a single node microk8s for testing, development, etc.

Reply:
Fixed